crypto/property/defn_cache.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2019, Oracle and/or its affiliates.  All rights reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#include <string.h>
+#include <openssl/err.h>
+#include <internal/propertyerr.h>
+#include <internal/property.h>
```

Comment:
Internal inclusions are normally quoted.

(I've had a discussion elsewhere that lead me to believe that we really should do so with the openssl/ inclusions as well, since we *are* including project local headers rather than system headers, but that's a matter we'd need to discuss on the project list)